spec/requests/host_initiator_groups_spec.rb

```diff
@@ -0,0 +1,112 @@
+describe "Host Initiator Groups API" do
+  context "POST /api/host_initiator_groups" do
+    it "with an invalid ems_id it responds with 404 Not Found" do
+      api_basic_authorize(collection_action_identifier(:host_initiator_groups, :create))
+
+      request = {
+        "action"   => "create",
+        "resource" => {
+          "ems_id"              => nil,
+          "name"                => "test_host_initiator_group",
+          "physical_storage_id" => "1",
+        }
+      }
+
+      post(api_host_initiator_groups_url, :params => request)
+
+      expect(response).to have_http_status(:bad_request)
+    end
+
+    it "creates new Host Initiator Group" do
+      api_basic_authorize(collection_action_identifier(:host_initiator_groups, :create))
+      provider = FactoryBot.create(:ems_autosde, :name => 'Autosde')
+      request = {
+        "action"   => "create",
+        "resource" => {
+          "ems_id"              => provider.id,
+          "name"                => "test_host_initiator_group",
+          "physical_storage_id" => "1",
```

Comment:
Prefer not to hard-code IDs in specs, can you `physical_storage = FactoryBot.create(:physical_storage)` and use `physical_storage.id.to_s` instead?